.eslintrc.js

```diff
@@ -38,11 +38,17 @@ module.exports = {
     "strict": [2, "global"],
     "prefer-const": 2,
     "curly": [2, "multi-line"],
+    "comma-dangle": [2, {
+      "arrays": "always-multiline",
+      "objects": "always-multiline",
+      "imports": "never",
+      "exports": "never",
+      "functions": "never"
```

Comment:
trailing commas in imports/exports/functions aren't used in lighthouse code, so I turned them off. (`function(a,b,)`?? Yuck)

Reply:
it's actually pretty nice for clean diffs and easy copy paste of multi-line function arguments :)

```
SomethingVeryLong.SomethingVeryLong = function somethingVeryLong(
  aLongName,
  aSecondLongName,
) {
 //
};
```

Reply:
Right, with "always-multiline" it would make sense 👍